appium/tests/helpers/TouchHelper.ts

```diff
@@ -17,12 +17,42 @@ class TouchHelper {
   }
 
   /**
-   * Dismiss popover by clicking popup outside
+   * Tap Screen
    */
-  static dismissPopover = async () => {
+  static tapScreen = async (position: ScreenPosition) => {
     const {width, height} = await driver.getWindowSize();
+    let pressOptions;
+    switch (position) {
+      case 'topLeft':
+        pressOptions = {x: 0, y: 0};
+        break;
```

Comment:
@sosnovsky could you please look at this PR / this method for a review - in case a simpler solution exists. Thanks!

Reply:
Current solution looks good for me 👍
The only possible improvement I see - using `enum` for `ScreenPosition`, so we'll have autocomplete for positions and it won't be possible to mistype them.

Reply:
@sosnovsky Thanks for your review.
By the way, currently it also supports autocompletion and it will throw error if user mistyped ScreenPosition.